[2024-11-29 11:17:17,819] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: kpi_calculation_pipeline.store_kpi 2024-11-22T16:05:00+00:00 [queued]>
[2024-11-29 11:17:17,920] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: kpi_calculation_pipeline.store_kpi 2024-11-22T16:05:00+00:00 [queued]>
[2024-11-29 11:17:17,922] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-11-29 11:17:17,923] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2024-11-29 11:17:17,924] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-11-29 11:17:18,012] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): store_kpi> on 2024-11-22T16:05:00+00:00
[2024-11-29 11:17:18,017] {standard_task_runner.py:52} INFO - Started process 202 to run task
[2024-11-29 11:17:18,022] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'kpi_calculation_pipeline', 'store_kpi', '2024-11-22T16:05:00+00:00', '--job-id', '376', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/data_injestion_dag/bazefield_kpi_calculation.py', '--cfg-path', '/tmp/tmp0svur5kf', '--error-file', '/tmp/tmpqsigdqte']
[2024-11-29 11:17:18,023] {standard_task_runner.py:77} INFO - Job 376: Subtask store_kpi
[2024-11-29 11:17:18,161] {logging_mixin.py:104} INFO - Running <TaskInstance: kpi_calculation_pipeline.store_kpi 2024-11-22T16:05:00+00:00 [running]> on host 3f28a2ef9091
[2024-11-29 11:17:18,392] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=kpi_calculation_pipeline
AIRFLOW_CTX_TASK_ID=store_kpi
AIRFLOW_CTX_EXECUTION_DATE=2024-11-22T16:05:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-11-22T16:05:00+00:00
[2024-11-29 11:17:18,449] {base.py:69} INFO - Using connection to: id: postgres_default. Host: 10.197.48.9, Port: 5432, Schema: kpi_data, Login: rakshit, Password: ***, extra: {}
[2024-11-29 11:17:18,623] {logging_mixin.py:104} INFO - successfully connected
[2024-11-29 11:17:18,707] {python.py:151} INFO - Done. Returned value was: None
[2024-11-29 11:17:18,730] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=kpi_calculation_pipeline, task_id=store_kpi, execution_date=20241122T160500, start_date=20241129T111717, end_date=20241129T111718
[2024-11-29 11:17:18,843] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-29 11:17:18,857] {local_task_job.py:151} INFO - Task exited with return code 0
